app/src/main/res/layout/restaurant_item.xml

```diff
@@ -16,144 +16,144 @@
+                        android:layout_height="46dp"
+                        android:contentDescription="@string/iconRestaurant"
+                        app:layout_constraintBottom_toBottomOf="parent"
+                        app:layout_constraintEnd_toEndOf="parent"
+                        app:layout_constraintHorizontal_bias="0.047"
                         app:layout_constraintStart_toStartOf="parent"
                         app:layout_constraintTop_toTopOf="parent"
-                        app:srcCompat="@drawable/coin" />
+                        app:layout_constraintVertical_bias="0.2"
+                        tools:srcCompat="@drawable/corral_icon" />
 
-                    <TextView
-                        android:id="@+id/textPrice"
-                        android:layout_width="wrap_content"
-                        android:layout_height="wrap_content"
-                        android:layout_marginStart="8dp"
-                        android:text="@string/restaurant_price"
-                        android:textColor="@color/material_dynamic_neutral_variant30"
-                        android:textSize="11sp"
-                        app:layout_constraintStart_toEndOf="@id/imageCoin2"
-                        app:layout_constraintTop_toTopOf="parent" />
 
                 </androidx.constraintlayout.widget.ConstraintLayout>
 
                 <androidx.constraintlayout.widget.ConstraintLayout
-                    android:layout_width="match_parent"
-                    android:layout_height="0dp"
-                    android:layout_weight="1">
+                    android:layout_width="210dp"
+                    android:layout_height="match_parent">
 
+                    <TextView
+                        android:id="@+id/textNameRestaurant"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:layout_marginTop="10dp"
+                        android:text="@string/r1"
+                        android:textColor="@color/black"
+                        android:textSize="12sp"
+                        android:textStyle="bold"
+                        app:layout_constraintStart_toStartOf="parent"
+                        app:layout_constraintTop_toTopOf="parent" />
 
                     <TextView
-                        android:id="@+id/textDistance"
-                        android:layout_width="wrap_content"
+                        android:id="@+id/textState1"
+                        android:layout_width="match_parent"
                         android:layout_height="wrap_content"
-                        android:layout_weight="1"
-                        android:text="@string/restaurant_distance"
-                        android:textColor="@color/material_dynamic_neutral20"
+                        android:text="@string/e1"
+                        android:textColor="@color/material_dynamic_neutral30"
```

Comment:
Use "@color/black" or another defined basic color instead of material_dynamic colors